webpack.config.js

```diff
@@ -0,0 +1,233 @@
+const path = require('path');
+const CleanWebpackPlugin = require('clean-webpack-plugin');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+const PeerDepsExternalsPlugin = require('peer-deps-externals-webpack-plugin');
+const postcssImport = require('postcss-import');
+const postcssPresetEnv = require('postcss-preset-env');
+const postcssReporter = require('postcss-reporter');
+const postcssCustomProperties = require('postcss-custom-properties');
+const postcssCustomMediaQueries = require('postcss-custom-media');
+const postcssPostcssColorModFunction = require('postcss-color-mod-function');
+
+const browserslist = {
+  development: ['chrome', 'firefox'].map(
+    browser => `last 2 ${browser} versions`
+  ),
+  production: ['>1%', 'not op_mini all', 'ie 11'],
+};
+
+const sourceFolders = [
+  path.resolve(__dirname),
+  path.resolve(__dirname, '../examples'),
+  path.resolve(__dirname, '../src'),
+];
+
+module.exports = {
+  entry: {
+    main: './src/index.js',
+    utils: './src/utils/index.js',
+  },
+  mode: 'production',
+  output: {
+    path: path.join(__dirname, 'dist'),
+    library: 'McUiKit',
+    libraryTarget: 'umd',
+    filename: chunkData =>
+      chunkData.chunk.name === 'main' ? 'index.js' : '[name]/index.js',
```

Comment:
This enables spitting out the nested build like

```
- dist
  - index.js
  - utils
    - index.js
```

We can adapt this function in case we want to export more things.

Note that the `filterDataAttributes` function will now exist twice. Once in the `main` bundle and once at this point. This isn't optional yet, as it increases the bundle size unnecessarily.

Reply:
Can you add this explanation to the code?